consensus/safety-rules/src/local_client.rs

```diff
@@ -77,10 +75,10 @@ impl TSafetyRules for LocalClient {
     fn sign_commit_vote(
         &mut self,
         ledger_info: LedgerInfoWithSignatures,
-        new_ledger_info: LedgerInfo,
+        last_block: ExecutedBlock,
```

Comment:
let's skip the extension check for this pr

Reply:
OK, I'll change it later.